core/src/main/java/org/elasticsearch/cluster/metadata/MetaDataIndexUpgradeService.java

```diff
@@ -150,7 +149,7 @@ private static boolean isSupportedVersion(IndexMetaData indexMetaData) {
                                     "index.translog.interval",
                                     "index.translog.sync_interval",
                                     "index.shard.inactive_time",
-                                    UnassignedInfo.INDEX_DELAYED_NODE_LEFT_TIMEOUT_SETTING));
+                                    "index.unassigned.node_left.delayed_timeout"));
```

Comment:
I removed all of this - it's a pre 2.0 BWC layer we don't need anymore in master